test/langtools/jdk/javadoc/doclet/testModules/TestModuleServicesLink.java

```diff
@@ -120,10 +122,10 @@ public void checkNavbarWithoutServices(Path base) throws Exception {
 
         checkOutput("m/module-summary.html", true,
                 """
-                    <li>Description&nbsp;|&nbsp;</li>
-                    <li>Modules&nbsp;|&nbsp;</li>
-                    <li><a href="#packages-summary">Packages</a>&nbsp;|&nbsp;</li>
-                    <li>Services</li>""");
+                    <ol class="toc-list">
+                    <li><a href="#" tabindex="0">Description</a></li>
+                    <li><a href="#packages-summary" tabindex="0">Packages</a></li>
+                    </ol>""");
```

Comment:
I had missed this subtlety earlier. The vertical bars have gone from the generated code. Nice!